.changes/unreleased/tinymce-TINY-11217-2024-09-10.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Added new default styles for list elements.
```

Comment:
```suggestion
body: Added new default `li` element styles to the content CSS to prevent child list elements from inheriting certain parent list item styles.
```